### PR TITLE
Refactor network selector and network data

### PR DESCRIPTION
**Pull request by sdlyy**
Resolves L2B-3040
Resolves L2B-3038

### CODE REVIEW

File: packages/frontend/src/view/components/NetworkSelector.tsx [hunk 1]
```diff
@@ -9,32 +20,103 @@ interface Props {
...
+  }, [focusRef, isActive])
+
+  return (
+    <div
+      ref={focusRef}
+      className={cx(
+        'flex min-w-fit cursor-pointer items-center justify-center gap-1 rounded py-3 pl-4 pr-6 text-center',
+        isActive
+          ? 'bg-yellow-100 text-black'
+          : 'bg-gray-300 text-white brightness-100 filter transition-all duration-300 hover:brightness-[130%]',
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/NetworkSelector.tsx:83]: Please tone down the hover brightness a tiny bit.
```suggestion
          : 'bg-gray-300 text-white brightness-100 filter transition-all duration-300 hover:brightness-[120%]',
```

File: packages/frontend/src/view/components/NetworkSelector.tsx [hunk 2]
```diff
@@ -9,32 +20,103 @@ interface Props {
...
+      )}
+      onClick={onClick}
+    >
+      <div className="flex h-[30px] w-[30px] items-center justify-center">
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/NetworkSelector.tsx:87]: Let's make the icon a bit smaller, similar to the designs
```suggestion
      <div className="flex h-6 w-6 items-center justify-center">
```

File: packages/frontend/src/view/components/NetworkSelector.tsx [hunk 3]
```diff
@@ -9,32 +20,103 @@ interface Props {
...
+
   return (
-    <section className="bg-gray-900 px-6 pb-0 pt-6">
-      <label className="mb-4 text-xs text-gray-500">Select network</label>
-      <Dropdown
-        defaultValue={defaultValue}
-        onChange={onChange}
-        options={options}
-      />
+    <section className="bg-gray-500 px-6 py-8 md:justify-center">
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/NetworkSelector.tsx:46]:
```suggestion
    <section className="bg-gray-500 p-6 md:justify-center">
```

File: packages/frontend/src/view/components/NetworkSelector.tsx [hunk 4]
```diff
@@ -9,32 +20,103 @@ interface Props {
...
+  icon: React.ReactNode
+  label: string
+  isActive: boolean
+  onClick: () => void
+}) {
+  const focusRef = useRef<HTMLDivElement>(null)
+
+  useEffect(() => {
+    if (focusRef.current && isActive) {
+      focusRef.current.scrollIntoView({ behavior: 'smooth', inline: 'center' })
```
Comment (michalsidzej) [inline packages/frontend/src/view/components/NetworkSelector.tsx:72]: Let's not scroll that aggressively, let's scroll the whole network selection section into view. 

https://github.com/l2beat/lz-monitoring/assets/38423054/cfc541c3-8b8f-4d5e-a313-5e9d17aacfab

File: packages/frontend/src/view/components/NetworkSelector.tsx (hunk 3 above)
Comment (michalsidzej) [reply packages/frontend/src/view/components/NetworkSelector.tsx:46]: Let's also reduce the padding on mobile. WYT about making it sticky to the top (just like on L2BEAT product page)
Screenshot 2023-11-21 at 11 42 41

Comment (sdlyy) [reply packages/frontend/src/view/components/NetworkSelector.tsx]: [feat: make network selector top-sticky](https://github.com/l2beat/lz-monitoring/pull/66/commits/61a5977e092da5b18b3c971dcca42136951a4daa)
[feat: refine selector paddings](https://github.com/l2beat/lz-monitoring/pull/66/commits/3dc4d905db10b1fc5de127cd6c95c32c30a4eb63)

File: packages/frontend/src/view/components/NetworkSelector.tsx (hunk 2 above)
Comment (sdlyy) [reply packages/frontend/src/view/components/NetworkSelector.tsx]: [feat: unify paddings across netowork pills](https://github.com/l2beat/lz-monitoring/pull/66/commits/b5e5778f86c4848569c84001a66a332050f9db31)

File: packages/frontend/src/view/components/NetworkSelector.tsx (hunk 1 above)
Comment (sdlyy) [reply packages/frontend/src/view/components/NetworkSelector.tsx]: [feat: update NetworkSelector pill hover brightness](https://github.com/l2beat/lz-monitoring/pull/66/commits/26cb0fac21a4f7cd03052f5d25f48ea7b17b6f22)

File: packages/frontend/src/view/components/NetworkSelector.tsx (hunk 4 above)
Comment (sdlyy) [reply packages/frontend/src/view/components/NetworkSelector.tsx]: https://github.com/l2beat/lz-monitoring/pull/66/commits/839833a1dcadd085daed104578f93228f467dae1

Comment (sdlyy) [reply packages/frontend/src/view/components/NetworkSelector.tsx]: Dropped vertical focus, this, together with sticky selector should do the trick

**Review by michalsidzej — APPROVED** [review]

**Pull request merged**